### DISCUSSION TITLE
New Help files

### DISCUSSION THREAD

**Discussion by bonzibudd** [Open-Shell/Open-Shell-Menu#1241] [General]
Wanted to make a quick post because I'm working on updating the Help files in order to be relevant to Open-Shell rather than Classic Shell. I'm also updating the very outdated design (here's how the new version will look similar to):
![image](https://user-images.githubusercontent.com/61938331/207441205-6246a494-66ea-44a6-b5ac-40f9dc60427c.png)

If anyone knows how to fix the "Can't reach this page" error when the viewer first opens, that would be helpful.

**Comment by bonzibudd** [comment]
Also question for @ge0rdi: Would it be appropriate to update the System Requirements section to include Windows 11, since it is now partially supported? Or would it be better to wait until more fixes are applied?

**Reply by ge0rdi** [reply]
I'd wait until more things are properly supported.

So far I've focused on start button and start menu.
There are some issues with taskbar customizations.
And `Classic Explorer` has issues too.

But really appreciate the effort with help file.
We even have #1173 issue for it. You can assign to you and eventually check the feedback there.

**Reply by ge0rdi** [reply]
#524 is also related to outdated help.

**Comment by ge0rdi** [comment]
> If anyone knows how to fix the "Can't reach this page" error when the viewer first opens, that would be helpful.

I have figured it out.
Each `Open-Shell` component tries to open its help sub-page. To identify what help page to open it uses component name as it is stored in registry.
For start menu component the page name should be `StartMenu.html`. But we have page named `Menu.html`. Apparently the bug was introduced during `Classic-Shell` to `Open-Shell` renaming and nobody just realized it.

Will fix it soon.
